index.html

```diff
@@ -0,0 +1,183 @@
+              <a href="#">
+                  <button class="mobile-nav-btn">Campaign</button>
+              </a>
+            </li>
+          </ul>
+        </nav> 
+      </header>
+    <section class="main-section">   
+        <div class="menu-btn">
+            <i class="fa-sharp fa-solid fa-bars"></i>
+        </div >
+     <div class="header-info">
+        <div class="summit-header">
+            <h4 class="header-h4">"Hello, Sharing World!"</h4>
+            <h1 class="header-h1">CREATIVE COMMONS<br>GLOBAL SUMMIT<br>2015</h1>
+        </div>
+        <p class="summit-paragraph">
+            A fun festival where people from more than 80 countries who believe in the value of sharing and openness and create positive change gather and share ideas will be held in Korea in October.
+        </p>
+        <h2 class="date">2015.2.10 (Mon) ~ 13 (Ths)</h2>
+        <p class="place location">
+          @ National Museum of Korea, Art Center Nabi, etc.</p>
+     </div>
+    </section>
+    <!--Program Section-->
+    <section class="program-section" id="program">
+        <h2 class="program-title">Main Program</h2>
+        <div class="red-underline"></div>
+        <div class="program-cards">
+            <div class="card">
+                <img class="program-image" src="./images/program_icon_01.png" alt="icon 1">
+                <h3 class="title">Lecture</h3>
+                <p class="program-info">Listen to the speakers from various countreis about their message for the young generations </p>
+            </div>
+            <div class="card">
+                <img class="program-image" src="./images/program_icon_02.png" alt="icon 2">
+                <h3 class="title">Lecture</h3>
+                <p class="program-info">Listen to the speakers from various countreis about their message for the young generations </p>
+            </div>
+            <div class="card">
+                <img class="program-image" src="./images/program_icon_03.png" alt="icon 3">
+                <h3 class="title">Lecture</h3>
+                <p class="program-info">Listen to the speakers from various countreis about their message for the young generations </p>
+            </div>
+            <div class="card">
+                <img class="program-image" src="./images/program_icon_04.png" alt="icon 4">
+                <h3 class="title">Lecture</h3>
+                <p class="program-info">Listen to the speakers from various countreis about their message for the young generations </p>
+            </div>
+            <div class="card">
+                <img class="program-image" src="./images/program_icon_05.png" alt="icon 5">
+                <h3 class="title">Lecture</h3>
+                <p class="program-info">Listen to the speakers from various countreis about their message for the young generations </p>
+            </div>
+        </div>
+        <div class="program-btn">
+            <a href="#">See The Whole Program</a>
+        </div>
```

Comment:
- [ optional ] For the mobile screen size I recommend you to adjust the Layout: composition and space between elements Just like the Design Template.
 The Design Template Look likes this : 
 
![Capture12](https://user-images.githubusercontent.com/93598189/191735305-b9a9b143-f5d2-4b80-9962-d75fde6532be.PNG)

Your Project Look likes this : 

![Capture1](https://user-images.githubusercontent.com/93598189/191735391-6707720b-2e98-48c3-8881-fa8143f5dce1.PNG)